cmd/convox/start.go

```diff
@@ -55,6 +57,29 @@ func cmdStart(c *cli.Context) error {
 		return stdcli.ExitError(errors.New("could not connect to docker daemon, is it installed and running?"))
 	}
 
+	if os.Getenv("PROVIDER") != "test" {
+		dockerVersionTest, err := docker.NewClientFromEnv()
+		if err != nil {
+			return stdcli.ExitError(err)
+		}
+
+		minDockerVersion, err := docker.NewAPIVersion("1.9")
+		e, err := dockerVersionTest.Version()
+		if err != nil {
+			return stdcli.ExitError(err)
+		}
+
+		currentVersionParts := strings.Split(e.Get("Version"), ".")
+		currentVersion, err := docker.NewAPIVersion(fmt.Sprintf("%s.%s", currentVersionParts[0], currentVersionParts[1]))
+		if err != nil {
+			return stdcli.ExitError(err)
+		}
+
+		if !(currentVersion.GreaterThanOrEqualTo(minDockerVersion)) {
+			return stdcli.ExitError(errors.New("You're version of docker is out of date (min: 1.9)"))
```

Comment:
https://docs.docker.com/v1.9/engine/misc/deprecated/ looks like it was actually deprecated in 1.8.

AWS is using 1.9 and there was only 3 months difference between the 1.8 and 1.9 release.

Happy to change to 1.8 if you prefer?